locale/lists/actionList/en.yaml

```diff
@@ -4,3 +4,12 @@ header:
     location: Location
     participants: Participants
     startTime: Time
+
```

Comment:
Some general rules of thumb when editing localization files:

* Don't include empty lines
* On each level (i.e. in each YAML object/dictionary), sort keys alphabetically.

This makes sure that the tooling which makes strings available to translators and converts them back to YAML does not create unnecessary diffs to the files.